### PR TITLE
it's not a set of a set of flags, it's just a set of flags

### PR DESCRIPTION
**Pull request by UnsungHero97**

### CODE REVIEW

File: conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt [hunk 1]
```diff
@@ -303,7 +303,7 @@ class OrganizationEntitySetsService(
                         propertyTypes.getValue(CONTACT).id to entitySet.contacts,
                         propertyTypes.getValue(DATASET_NAME).id to setOf(entitySet.name),
                         propertyTypes.getValue(DESCRIPTION).id to setOf(entitySet.description),
-                        propertyTypes.getValue(FLAGS).id to setOf(entitySet.flags),
+                        propertyTypes.getValue(FLAGS).id to entitySet.flags.map { flag -> flag.name }.toSet(),
```
Comment (katherinebernstein) [inline conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt:306]: why not just `entitySet.flags.toSet()` ?

**Review by geekbeast — APPROVED** [review]

File: conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt (hunk 1 above)
Comment (geekbeast) [reply conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt:306]: +1

Comment (UnsungHero97) [reply conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt:306]: I'll get you a stacktrace.

Comment (UnsungHero97) [reply conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt:306]:
```
java.lang.IllegalStateException: Unable to write to property type c0451cbf-3d66-41cf-9253-ea6f9bb38e16 with values [METADATA]: Entity set f09c25f7-7c76-488a-98b8-f06e38db6014 with entity key id 00000000-0000-0000-8000-000000000000
        at com.openlattice.postgres.JsonDeserializer$Companion.validateFormatAndNormalize(JsonDeserializer.kt:94) ~[conductor-client-2021.jar:?]
        at com.openlattice.data.storage.PostgresEntityDataQueryService.upsertEntities(PostgresEntityDataQueryService.kt:304) ~[conductor-client-2021.jar:?]
        at com.openlattice.data.storage.PostgresEntityDataQueryService.upsertEntities$default(PostgresEntityDataQueryService.kt:290) ~[conductor-client-2021.jar:?]
        at com.openlattice.data.storage.PostgresEntityDataQueryService.partialReplaceEntities(PostgresEntityDataQueryService.kt:506) ~[conductor-client-2021.jar:?]
        at com.openlattice.data.storage.PostgresEntityDataQueryService$$FastClassBySpringCGLIB$$2c77ccfb.invoke(<generated>) ~[conductor-client-2021.jar:?]
        ...
Caused by: java.lang.IllegalStateException: Expected string for property type String with data c0451cbf-3d66-41cf-9253-ea6f9bb38e16,  received class com.openlattice.edm.set.EntitySetFlag
        at com.google.common.base.Preconditions.checkState(Preconditions.java:843) ~[guava-27.1-jre.jar:?]
        at com.openlattice.postgres.JsonDeserializer$Companion.validateFormatAndNormalize(JsonDeserializer.kt:200) ~[conductor-client-2021.jar:?]
        at com.openlattice.postgres.JsonDeserializer$Companion.validateFormatAndNormalize(JsonDeserializer.kt:79) ~[conductor-client-2021.jar:?]
        ... 56 more
```

Comment (katherinebernstein) [reply conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt:306]: ahh makes sense

**Review by katherinebernstein — APPROVED** [review]

**Pull request merged**